### PR TITLE
OSSM-8432: 2.6.4 (At Stage): [DOC] Release Notes

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc update; OSSM 2.6.4 (At Stage): Release Notes

Doc JIRA: https://issues.redhat.com/browse/OSSM-8432

Fix Version: 4.14+

Doc Preview: https://85846--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes#ossm-release-2-6-4_ossm-release-notes

QE Review: @unsortedhashsets 
Peer Review: @lahinson

### CODE REVIEW

**Review by unsortedhashsets — APPROVED** [review]

File: modules/ossm-release-2-6-4.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+[id="ossm-release-2-6-4_{context}"]
+= {SMProductName} version 2.6.4
+
+This release of {SMProductName} updates the {SMProductName} Operator version to 2.6.4, and includes the following `ServiceMeshControlPlane` resource version updates: 2.6.4, 2.5.7, and 2.4.13.
+
+This release addresses Common Vulnerabilities and Exposures (CVEs) and is supported on {product-title} 4.14 and later.
+
+include::snippets/ossm-current-version-support-snippet.adoc[]
+
+The most current version of the {KialiProduct} can be used with all supported versions of {SMProductName}. The version of {SMProductShortName} is specified by using the `ServiceMeshControlPlane` resource. The version of {SMProductShortName} automatically ensures a compatible version of Kiali.
```
Comment (lahinson) [inline modules/ossm-release-2-6-4.adoc:16]:
```suggestion
The most current version of the {KialiProduct} can be used with all supported versions of {SMProductName}. The version of {SMProductShortName} is specified by using the `ServiceMeshControlPlane` resource. The version of {SMProductShortName} automatically ensures a compatible version of {KialiProduct}.
```
I'm not sure what the precedent is for referring to the Kiali Operator, but typically, the word "Operator" is included whenever an Operator is being referred to.

File: modules/ossm-release-2-4-13.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-4-13_{context}"]
+= {SMProductName} version 2.4.13
+
+This release of {SMProductName} is included with the {SMProductName} Operator 2.6.4, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.14 and later.
```
Comment (lahinson) [inline modules/ossm-release-2-4-13.adoc:10]:
```suggestion
This release of {SMProductName} is included with the {SMProductName} Operator 2.6.4 and is supported on {product-title} 4.14 and later. This release addresses Common Vulnerabilities and Exposures (CVEs).
```
I divided the sentence in two to make it a little easier to read.

File: modules/ossm-release-2-5-7.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-5-7_{context}"]
+= {SMProductName} version 2.5.7
+
+This release of {SMProductName} is included with the {SMProductName} Operator 2.6.4, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.14 and later.
+
```
Comment (lahinson) [inline modules/ossm-release-2-5-7.adoc:11]:
```suggestion
```

File: modules/ossm-release-2-5-7.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-5-7_{context}"]
+= {SMProductName} version 2.5.7
+
+This release of {SMProductName} is included with the {SMProductName} Operator 2.6.4, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.14 and later.
```
Comment (lahinson) [inline modules/ossm-release-2-5-7.adoc:10]:
```suggestion
This release of {SMProductName} is included with the {SMProductName} Operator 2.6.4 and is supported on {product-title} 4.14 and later. This release addresses Common Vulnerabilities and Exposures (CVEs).
```

**Review by lahinson — COMMENTED** [review]
Looks good! Just a few minor items for your consideration.

File: modules/ossm-release-2-6-4.adoc (hunk 1 above)
Comment (shreyasiddhartha) [reply modules/ossm-release-2-6-4.adoc:16]: @lahinson, I am ignoring this comment to maintain consistency with the previous releases. We use a similar template for the past z-streams.

**Review by unsortedhashsets — APPROVED** [review]

**Review by ShaunaDiaz — COMMENTED** [review]
LGTM

**Comment by unsortedhashsets** [comment]
lgtm

**Comment by ShaunaDiaz** [comment]
> @ShaunaDiaz: #85846 failed to apply on top of branch "enterprise-4.13":
> 
> ```
> Applying: OSSM-8432
> Using index info to reconstruct a base tree...
> M	_attributes/common-attributes.adoc
> A	modules/ossm-release-2-6-3.adoc
> M	service_mesh/v2x/servicemesh-release-notes.adoc
> Falling back to patching base and 3-way merge...
> Auto-merging service_mesh/v2x/servicemesh-release-notes.adoc
> CONFLICT (content): Merge conflict in service_mesh/v2x/servicemesh-release-notes.adoc
> CONFLICT (modify/delete): modules/ossm-release-2-6-3.adoc deleted in HEAD and modified in OSSM-8432. Version OSSM-8432 of modules/ossm-release-2-6-3.adoc left in tree.
> Auto-merging _attributes/common-attributes.adoc
> CONFLICT (content): Merge conflict in _attributes/common-attributes.adoc
> error: Failed to merge in the changes.
> hint: Use 'git am --show-current-patch=diff' to see the failed patch
> hint: When you have resolved this problem, run "git am --continue".
> hint: If you prefer to skip this patch, run "git am --skip" instead.
> hint: To restore the original branch and stop patching, run "git am --abort".
> hint: Disable this message with "git config advice.mergeConflict false"
> Patch failed at 0001 OSSM-8432
> ```

@shreyasiddhartha Please do a manual CP for this one.

**Comment by ShaunaDiaz** [comment]
@shreyasiddhartha My understanding is that everything should go back to 4.12 with no skip branches. 4.10 is out of support and no longer updated.

**Comment by shreyasiddhartha** [comment]
@ShaunaDiaz In my PR, I have mentioned only 4.14+. I think that's already done. 4.10 was label was added by a peer reviewer, which isn't required for this PR. Anyway this PR cannot be applied before 4.14, as there were multiple changes to docs with no updates to the previous z-stream. IMHO, 4.14+ is enough.

**Pull request merged**